lib/pbench/server/api/resources/users_api.py

```diff
@@ -269,14 +268,22 @@ def post(self):
                         "message": "failure message"
                     }
         """
-        auth_token = self.auth.token_auth.get_auth()["token"]
-        user = self.auth.token_auth.current_user()
+        auth_token = self.auth.get_auth_token(self.logger)
```

Comment:
So this only fetches the `auth_token` from the API call, right?  It does not look the token up in the database?